src/modules/accounting/accounting.py

```diff
@@ -224,11 +225,28 @@ def get_rebase_after_report(self, blockstamp: ReferenceBlockStamp, cl_only=False
 
         return LidoReportRebase(*result)
 
+    def _get_slots_elapsed_from_last_report(self, blockstamp: ReferenceBlockStamp):
+        """If no report was finalized return slots elapsed from initial epoch from contract"""
+        chain_conf = self.get_chain_config(blockstamp)
+        frame_config = self.get_frame_config(blockstamp)
+
+        last_ref_slot = self._get_processing_ref_slot(blockstamp)
+
+        if last_ref_slot:
+            slots_elapsed = blockstamp.ref_slot - last_ref_slot
+        else:
+            slots_elapsed = blockstamp.ref_slot - frame_config.initial_epoch * chain_conf.slots_per_epoch
+
+        return slots_elapsed
+
+    def _get_processing_ref_slot(self, blockstamp: BlockStamp):
+        return self.report_contract.functions.getLastProcessingRefSlot().call(block_identifier=blockstamp.block_hash)
+
     @lru_cache(maxsize=1)
     def _is_bunker(self, blockstamp: ReferenceBlockStamp) -> bool:
         frame_config = self.get_frame_config(blockstamp)
         chain_config = self.get_chain_config(blockstamp)
-        cl_rebase_report = self.get_rebase_after_report(blockstamp, cl_only=True)
+        cl_rebase_report = self.get_rebase_after_report(blockstamp, ignore_execution_rewards=True)
```

Comment:
A flag that changes the behavior of a function is not a good practice. I would suggest splitting it into several, like: `get_full_rebase_after_report` and `get_cl_rebase_after_report`

Reply:
These two functions inside will call the same function with the same flag. Not sure if we need such abstraction.
Otherwise, we violate DRY.